include/config/battle.h

```diff
@@ -8,6 +8,8 @@
 #define B_CONFUSION_SELF_DMG_CHANCE GEN_LATEST // In Gen7+, confusion has a 33.3% of self-damage, instead of 50%.
 #define B_MULTI_HIT_CHANCE          GEN_LATEST // In Gen5+, multi-hit moves have different %. See Cmd_setmultihitcounter for values.
 #define B_WHITEOUT_MONEY            GEN_LATEST // In Gen4+, the amount of money lost by losing a battle is determined by the amount of badges earned. Previously, it would cut the current money by half. (While this change was also in FRLG, for the sake of simplicity, setting this to GEN_3 will result in RSE behavior.)
+#define B_LIGHT_BALL_ATTACK_BOOST  GEN_LATEST // In Gen4+, Light Ball multiplies the base power of physical moves by 2.
+#define B_SANDSTORM_SPDEF_BOOST    GEN_LATEST // In Gen4+, Sandstorm multiplies the Special Defense of Rock types by 1.5.
```

Comment:
```suggestion
#define B_LIGHT_BALL_ATTACK_BOOST   GEN_LATEST // In Gen4+, Light Ball multiplies the base power of physical moves by 2.
#define B_SANDSTORM_SPDEF_BOOST     GEN_LATEST // In Gen4+, Sandstorm multiplies the Special Defense of Rock types by 1.5.
```

Reply:
I'm pretty sure I did this, already? Why is saying there's still changes to be made

Reply:
Dunno. When you apply the changes requested, you can just hit the "Resolve conversation" button though.